protocols/swupdate/Readme.md

```diff
@@ -0,0 +1,7 @@
+This is basically a complete copy of the cosi protocol (created by 
+@nikkolasg) with some nice additional features (possibility to pass the 
+message directly in the Announcement and the possibility to validate 
+this data in a general way). 
+This shouldn't be merged into master as it is. All changes (which are 
+useful) should most probably directly go into the main cosi 
```

Comment:
+1. That should be in our "next steps" list (somewhere in our minds). So let's remove this protocol from this PR. I make an issue on `dedis/cosi`.